CMSgov/easi-app
# Correct argument order when calling function to send LCID extension emails

No ticket, bug [reported on Slack](https://cmsgov.slack.com/archives/C0305U1014Y/p1666798763341749) by Matt Schmid from the governance team.

## Changes and Description

Correct the order of the `projectName` and `requester` arguments to match what's expected by the email client methods. From `pkg/email/extend_lcid.go` (note that `projectName` comes _before_ `requester`):
```go
func (c Client) SendExtendLCIDEmailToMultipleRecipients(
	ctx context.Context,
	recipients models.EmailNotificationRecipients,
	systemIntakeID uuid.UUID,
	projectName string,
	requester string,
	newExpiresAt *time.Time,
	newScope string,
	newNextSteps string,
	newCostBaseline string,
) error {
  // implementation...
}
```

## How to test this change

1. Connect to the VPN.
2. Comment out the lines in `pkg/server/routes.go` that use the local fake CEDAR LDAP client.
3. Run the application locally.
4. Put in a GRT request.
5. Issue an LCID for the request.
6. Extend the LCID for the request.
7. Verify that the email notification for LCID extension has the project name and requester in the right positions.

Email with the incorrect order:

![lcid extension email - incorrect argument order](https://user-images.githubusercontent.com/92891039/198081073-7a869a34-a096-415a-9ac4-4bcd252af1ac.PNG)

Email with the fixed order:

![lcid extension email - fixed argument order](https://user-images.githubusercontent.com/92891039/198081114-a3dfbd11-6518-4d52-a850-e49c34b7e205.PNG)
